client/packages/invoices/src/Prescriptions/DetailView/History/columns.ts

```diff
@@ -4,12 +4,12 @@ import { HistoryItem } from './HistoryModal';
 export const useHistoryColumns = () => {
   const columns: ColumnDescription<HistoryItem>[] = [
     {
-      label: 'label.qty-item-name',
-      key: 'qtyItemName',
-      accessor: ({ rowData }) => {
-        const { unitQuantity, itemName } = rowData;
-        return `${unitQuantity}, ${itemName}`;
-      },
+      label: 'report.item-name',
+      key: 'itemName',
+    },
+    {
+      label: 'label.quantity',
```

Comment:
Can you please change the label to say "Unit quantity" -- we're trying to always be explicity about whether we're talking in units or packs, as just "quantity" is ambigous and been a common source of confusion and bugs.